client/src/components/TopNav.jsx

```diff
@@ -0,0 +1,190 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import axios from 'axios';
+import DropDown from './DropDown';
+import Friends from './Friends';
+import Locations from './Locations'
+const Styled = require('./Styles');
+
+// tried to implement styles in a separate file but it isn't working because it says it isn't "returning a string"
+
+const TopNav = (props) => {
+
+  const [text, setText] = useState('');
+  const [manageFriends, setManageFriends] = useState(false);
```

Comment:
`manageFriends` sounds like it's performing an action, but the value is a boolean. Maybe you could rename it to be more obviously a boolean? E.g. `shouldManageFriends`?